src/raftstore/store/store.rs

```diff
@@ -586,9 +589,10 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             .next() {
             let exist_region = self.region_peers[&exist_region_id].region();
             if enc_start_key(exist_region) < data_end_key(msg.get_end_key()) {
-                debug!("msg {:?} is overlapped with region {:?}, ignored",
+                debug!("msg {:?} is overlapped with region {:?}, queue for later split",
```

Comment:
sorry, I am wrong, I meet the condition here https://github.com/pingcap/tikv/blob/48d44fdd861a94f7aed0774f85a6645b046f7fbd/src/raftstore/store/store.rs#L717

```
2017/03/09 17:46:49.962 store.rs:650: [INFO] [region 24] handle raft message MsgRequestVote, from 25 to 26
2017/03/09 17:46:49.962 store.rs:1943: [ERROR] [store 3] handle raft message err: Other(StringError("[region 24] region not exist but not tombstone: state: Normal region {id: 24 start_key: \"t\\200\\000\\000\\000\\000\\000\\000\\377\\023_r\\200\\000\\000\\000\\000\\377\\0004\\370\\000\\000\\000\\000\\000\\372\" end_key: \"\" region_epoch {conf_ver: 3 version: 5} peers {id: 25 store_id: 1} peers {id: 26 store_id: 3} peers {id: 27 store_id: 2}}"))
```